Arguments needed to smoothly build modules without further config by the user

Details:
 - gst-plugins-bad: might try to enable vulkan extension when
                    some very common packages are installed,
                    but vulkan-headers is not.

diff --git a/jhbuild/defaults.jhbuildrc b/jhbuild/defaults.jhbuildrc
index 6820d994..a41f4ba5 100644
--- a/jhbuild/defaults.jhbuildrc
+++ b/jhbuild/defaults.jhbuildrc
@@ -93,7 +93,9 @@ svnroots = {}
 branches = {}
 module_autogenargs = {}
 module_cmakeargs = {}
-module_mesonargs = {}
+module_mesonargs = {
+    'gst-plugins-bad': '-D vulkan=disabled',
+}
 module_makeargs = {}
 module_ninjaargs = {}
 module_extra_env = {}
